lib/archethic/mining/pending_transaction_validation.ex

```diff
@@ -137,8 +137,8 @@ defmodule Archethic.Mining.PendingTransactionValidation do
               key_certificate,
               root_ca_public_key
             )},
-         {:conn, true} <-
-           {:conn, valid_connection?(ip, port, previous_public_key, should_validate_node_ip?())} do
+         {:conn, :ok} <-
+           {:conn, valid_connection?(ip, port, previous_public_key)} do
```

Comment:
if you return an ok tuple, better to no use `?` at the end of the function, but maybe rename it as`validate_connectioon`

Reply:
agreed